staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition/etcd.go

```diff
@@ -57,6 +57,14 @@ func NewREST(scheme *runtime.Scheme, optsGetter generic.RESTOptionsGetter) *REST
 	return &REST{store}
 }
 
+// Implement CategoriesProvider
+var _ rest.CategoriesProvider = &REST{}
+
+// Categories implements the CategoriesProvider interface. Returns a list of categories a resource is part of.
+func (r *REST) Categories() []string {
+	return []string{"all"}
```

Comment:
`all` is for user facing resources.  CRDs are highly privileged.  They don't belong in the all list.

Reply:
Right, came across https://github.com/kubernetes/community/blob/master/contributors/devel/kubectl-conventions.md#rules-for-extending-special-resource-alias---all now. 👍 

/cc @liggitt since you mentioned about this on slack.

Reply:
> @deads2k:  I could see allowing a user to specify their categories.

that is what I meant